web/src/store/publish.js

```diff
@@ -0,0 +1,28 @@
+import { publishRest } from '@/rest';
+
+export default {
+  namespaced: true,
+  state: {
+    versions: null,
```

Comment:
I don't think any of this should be in Vuex at all. Since every different navigation of the `DandisetLandingView` is going to be overwriting all of the state here, it's better to keep it within the `DandisetLandingView` itself, rather than deal with the extra complexity and potential desynchronization bugs of having it in a singleton Vuex module.

Reply:
I agree, and if `DandisetLandingView` is the only place using this it makes sense. I know we plan on showing version info in the main dandiset listing component ([figma link](https://www.figma.com/proto/kEHOgqU0rcOIyCYIDlpSMV/DANDI-UI?scaling=min-zoom&node-id=16%3A78)), but that seems orthogonal to this, and depends on the specifics of how we want display that info.